src/SBP_operators.jl

```diff
@@ -639,7 +639,7 @@ the quadrature rule associated with the SBP derivative operator `D`.
 """
 function integrate(func, u::AbstractVector, D::DerivativeOperator)
```

Comment:
You are right. However, I do not think that this is extremely performance critical (in the sense that everybody will want to remove the boundscheck via `@inbounds`) since it operates at a global level. Even if people wanted to remove it, they could use `@inline` at the call site. For me, it is just a way to mark bounds checks explicitly in the code to make them easier to find/spot.

Reply:
Ok, I see. Thanks for the clarification.